JSDemos/Demos/RadioGroup/Overview/description.md

```diff
@@ -1 +1,17 @@
-The RadioGroup is&nbsp;a&nbsp;UI component that contains a&nbsp;set of&nbsp;radio buttons and allows an&nbsp;end user to&nbsp;make a&nbsp;single selection from the set. This demo illustrates how to&nbsp;change the component layout, define a&nbsp;custom template and handle the value change event.
\ No newline at end of file
+The RadioGroup component contains a set of radio buttons and allows users to make a single selection from the set. This demo illustrates how to bind component to data, change the component layout, define a custom template, and handle the value change event.
+
+## Bind RadioGroup to Data
+
+You can display RadioGroup items from the [items](/Documentation/ApiReference/UI_Components/dxRadioGroup/Configuration/items/) array or a [dataSource](/Documentation/ApiReference/UI_Components/dxRadioGroup/Configuration/#dataSource). If your data is an array of objects, use the [displayExpr](/Documentation/ApiReference/UI_Components/dxRadioGroup/Configuration/#displayExpr) and the [valueExpr](/Documentation/ApiReference/UI_Components/dxRadioGroup/Configuration/#valueExpr) properties. [displayExpr](/Documentation/ApiReference/UI_Components/dxRadioGroup/Configuration/#displayExpr) specifies a data source field that supplies texts for buttons. [valueExpr](/Documentation/ApiReference/UI_Components/dxRadioGroup/Configuration/#valueExpr) specifies a data source field that supplies values to the value property when users select a button. Leave [valueExpr](/Documentation/ApiReference/UI_Components/dxRadioGroup/Configuration/#valueExpr) unspecified if you need to supply the entire data object to the value property.
+
+## Configure the Layout
+
+The RadioGroup supports horizontal (default for tablets) and vertical (default for other devices) layouts. To change the layout for all types of devices, specify the [layout](/Documentation/ApiReference/UI_Components/dxRadioGroup/Configuration/#layout) property.
+
+## Customize Item Appearance
```

Comment:
```suggestion
## Customize the Item Appearance
```

Reply:
I've heard that the title rules are a bit different and you can omit the articles. I tried to google it, but only found [this article](https://en.wikipedia.org/wiki/Headline#Headlinese). And I've been omitting some articles in the headings and article titles for a pretty long time and haven't even questioned it. The question is: _do we have any guidlines or rules about the titles?_ I found only the title case rule and "how to name the article" in the guidelines, but there's nothing about the grammar.

Reply:
Alas, no, we do not have one yet. However, I prefer to follow the same rules we use for the rest of our texts. Either way, the thing you did here does not look consistent. First you say in a title:

> Configure **the** Layout

Then, you omit an article in the next title:

> Customize Item Appearance

The Item Appearance literally means "the appearance of all items in general", therefore, it suits the header well.

Reply:
Okay, got you!